src/renderers/WebGLRenderer.js

```diff
@@ -2512,6 +2512,12 @@ function WebGLRenderer( parameters ) {
 
 		var framebuffer = properties.get( renderTarget ).__webglFramebuffer;
 
+		if ( renderTarget.isWebGLRenderTargetCube && activeCubeFaceIndex != undefined ) {
```

Comment:
Minor code style issue:
```js
if ( renderTarget.isWebGLRenderTargetCube && activeCubeFaceIndex !== undefined ) {
```

Reply:
Yeah sure, thanks.

Reply:
```suggestion
		if ( renderTarget.isWebGLRenderTargetCube && activeCubeFaceIndex !== undefined ) {
```

Reply:
Sry, I don't have the permission to commit the suggestion.

Reply:
Well, this is my first pull request^^
I committed the change...